css/index.css

```diff
@@ -143,4 +143,64 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+.top-of-page {
+	width: 800px;
+	margin: 0 auto;
+}
+
+header {
+	display: flex;
+	justify-content: space-between;
+	align-items: baseline;
+	margin: 26px 0;
+}
+
+header nav a {
+margin: 0 38px;
+}
+
+.top-of-page img {
+	margin-bottom: 0;
+}
+
+nav a {
+	text-decoration: none;
+	color:black;
+}
+
+.about-page-intro {
+	border-bottom:1px dashed black;
+	padding-bottom: 23px;
+	margin-bottom: 38px;
+}
+
+.mid-section {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}
+
+.mid-content {
+	width: 46%;
+	border-bottom:1px dashed black;
+	margin-bottom: 50px;
+}
+
+button {
+	margin: 20px 0;
+	border-radius: 10px;
+	background-color: white;
+	color: black;
+	padding: 5px 22px;
+	border:2px solid black;
+}
```

Comment:
you can add in the cursor: pointer property on the buttons to indicate to the users that they are to be clickable